.github/workflows/build_test.yml

```
@@ -0,0 +1,112 @@
+name: Standard Build and Test
+
+on:
+  # allows us to run workflows manually
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - develop
+      - master
+  push:
+    branches:
+      - develop
+      - master
+
+jobs:
+  Housekeeping:
+    runs-on: ubuntu-latest
+    container:
+      image: svalinn/dagmc-ci-ubuntu-18.04-housekeeping:latest
```

Should this point to the ghcr image instead of the dockerhub one?
 
```suggestion
      image: ghcr.io/svalinn/dagmc-ci-ubuntu-18.04-housekeeping:latest
```

---

One step at a time?? or take the plunge??  I guess we have CI to test it, so I agree we go for it

---

Oh I think I made bad choice here https://github.com/svalinn/DAGMC/pull/748/commits/906afc41ba0d6270f636225d9a27b68e6764698c. 

I was building both housekeeping images (16.04 and 18.04) then I reduced it down to just one and I chose 16.04 for some unknown reason. So the 18.04 image is there but it is just not kept up to date like the 16.04 house keeping image